src/admin-dhis2/admin-dhis2.service.js

```diff
@@ -48,13 +48,18 @@
             getServerDatasets: {
                 url: openlmisUrlFactory('/api/serverConfiguration/:id/datasets'),
                 method: 'GET'
+            },
+            removeDataset: {
+                url: openlmisUrlFactory('/api/serverConfiguration/:serverId/datasets/:datasetId'),
+                method: 'DELETE'
             }
         });
 
         this.getServerConfig = getServerConfig;
         this.addServer = addServer;
         this.editServer = editServer;
         this.removeServer = removeServer;
+        this.removeDataset = removeDataset;
```

Comment:
please move it under `this.getServerDatasets = getServerDatasets;` we should seperate the server and dataset endpoints

Reply:
ok, updated